src/main/java/com/fvv/bookstore/bean/Cellphone.java

```diff
@@ -12,55 +12,31 @@
  */
 public class Cellphone extends Hardware implements Serializable {
 
-	private static final long serialVersionUID = 1L;
-	private Double screenSize;
+	private static final long serialVersionUID = 1L;	
 	private Integer storageMemory;
 	private Integer camera;
```

Comment:
if this represents pixels, the name should be camPixels

Reply:
Ok